mod_bot/main.py

```diff
@@ -88,7 +90,8 @@ def add_minutes(hstart, mstart, mn):
             
         if words[0] == "!raid":
             isOk = True
-            
+
+            words = clean_raid_command(words)
```

Comment:
Error coming from here : the call to the clean method should be before testing if first word is "!raid"